basicLibrary/src/test/java/basicLibrary/LibraryTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basicLibrary;
+
+import org.junit.Test;
+
+import static basicLibrary.Library.*;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
+        Library classUnderTest = new Library();
+        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+    }
+
+    @Test
+    public void testRoll(){
+        assertArrayEqual("Six sided die is rolled, random value must be printed", new int[] {3,2,1,4}, roll(4));
```

Comment:
In your tests you need to be sure to use assert statements that are part of JUnit, unfortunately creating your own doesn't work. If you change the expected value this test will still pass because the assert method your wrote doesn't do any type of comparison!